va/va.go

```diff
@@ -22,6 +24,7 @@ import (
 	"github.com/letsencrypt/pebble/acme"
 	"github.com/letsencrypt/pebble/ca"
 	"github.com/letsencrypt/pebble/core"
+	"github.com/miekg/dns"
```

Comment:
I didn't know it was available :-) I was following the Boulder VA which used this dep. I'll try to rework on top of `net.LookupTXT`